tests/unit/ValidationUtilsTest.ts

```diff
@@ -307,9 +307,9 @@ describe('ValidationUtils', () => {
         });
 
         test('Invalid account route', () => {
-            expect(ValidationUtils.isValidAccountRoute(undefined)).toBe(false);
+            expect(ValidationUtils.isValidAccountRoute(undefined as unknown as number)).toBe(false);
             expect(ValidationUtils.isValidAccountRoute(0)).toBe(false);
-            expect(ValidationUtils.isValidAccountRoute('123aaa')).toBe(false);
+            expect(ValidationUtils.isValidAccountRoute('123aaa' as unknown as number)).toBe(false);
         });
```

Comment:
I think that test case can be removed because its handled by TS? WDYT?

Reply:
I agree